projects/storefrontlib/src/cms-components/checkout/components/shipping-address/address-form/address-form.component.ts

```diff
@@ -166,6 +165,16 @@ export class AddressFormComponent implements OnInit, OnDestroy {
       if (this.addressData.region) {
         this.regionSelected(this.addressData.region);
       }
+    } else {
+      this.userService
+        .get()
+        .subscribe(data => {
+          this.address.patchValue({
+            firstName: data.firstName,
+            lastName: data.lastName,
+          });
+        })
+        .unsubscribe();
```

Comment:
It's generally more preferable to use `.take(1)` than subbing and unsubbing. Please change that here